src/neural/network_blas.cc

```diff
@@ -188,8 +188,11 @@ class BlasNetwork : public Network {
  public:
   virtual ~BlasNetwork(){};
 
-  BlasNetwork(const Weights& weights, const OptionsDict& /* options */)
+  BlasNetwork(const Weights& weights, const OptionsDict& options)
       : weights_(weights) {
+    bool verbose = options.GetOrDefault<bool>("verbose", false);
```

Comment:
I think I'd prefer this to be true by default, explicit is better than implicit, and letting users know exactly which backend is currently running is very useful imo

Reply:
Are you sure? I also fear that it can disturb UCI clients.

Reply:
nah lczero has been outputting similar info and it hasn't ever caused problems

Reply:
Outputting to stdio is surely bad for uci. But to stderr is fine.